packages/transaction-builder/src/index.ts

```diff
@@ -1 +1,204 @@
-export {}
+import { Buffer } from 'buffer'
+import {
+  MsgProtoAppStake,
+  MsgProtoAppUnstake,
+  MsgProtoNodeStakeTx,
+  MsgProtoNodeUnjail,
+  MsgProtoNodeUnstake,
+  MsgProtoSend,
+} from './models/msgs'
+import { AbstractProvider } from '@pokt-foundation/pocketjs-abstract-provider'
+import { KeyManager } from '@pokt-foundation/pocketjs-signer'
+import {
+  RawTxRequest,
+  TransactionResponse,
+} from '@pokt-foundation/pocketjs-types'
+import { TxEncoderFactory } from './factory/tx-encoder-factory'
+import { TxMsg, CoinDenom, TxSignature } from './models/'
+
+export type ChainID = 'mainnet' | 'testnet'
```

Comment:
I think we could add `localnet` here, as it's usually called like that on the `e2e-stack`

Reply:
Good catch!

Reply:
Done in https://github.com/pokt-foundation/pocket-js-slim/pull/39/commits/79ea05f1f9689bcc2e6546d6c1fb421610503b55